legacy update (#661)
AdguardTeam/KnowledgeBase: docs/general/license/legacy.md

Standard were valid only for desktops, and Mobile were only for smartphones and tablets.

> Premium licenses could have been used on 2-18 devices. Depending on the number of devices selected at the time of purchase, you could activate AdGuard on the same number of computers and mobile devices at the same time: 2+2, 3+3, etc.

Reviewer: не очень понимаю смысл правки

Reviewer: ага

> For example, you can have a Premium 2+2 device licence, which means the licence can be installed on 2 computers and 2 phones/tablets, or a 9+9 device licence (18 devices in total), which means the licence can be installed on 9 computers and 9 phones/tablets.

Reviewer: license
и тоже не понимаю остальные исправления

For all types (Standard, Mobile, Premium) there were options for a Yearly and Lifetime license.

In October 2019, AdGuard switched to a new licensing system: **Personal** licences for up to 3 devices or **Family** licences for up to 9 devices. Both licence types can be purchased early or for life.

Another change is that the **new types of licence keys can now be used on any device, in any combination**. For example, a Personal licence can be used on Android, Mac and iOS, as well as on two Windows computers and an Apple tablet.

## What happened to the legacy licenses?

**Standard licenses** were automatically made **universal**, which means that they can be activated on any device.

**Mobile licenses** remained unchanged and platform-limited (they can still only be used on phones or tablets).

**Premium licenses** also remained unchanged and platform-limited. And although **Premium licenses are now displayed in your personal account by the total number of devices**, they can still be activated on desktop + mobile set of devices.

For example, this is how the 1+1 key looks like:

> ![1+1 key](https://cdn.adtidy.org/blog/new/ffb1pjlegacy1.png)

Reviewer: используй моноснап для того, чтобы заблюрить лицензионный ключ, пожалуйста. так, как это сделано на предыдущем скрине

*But despite the total number of devices (2) it can be activated on 1 desktop and 1 mobile device.*

>All the outdated licenses have a "legacy" badge in the AdGuard account. And they are automatically converted to a new type when you either renew or upgrade them. But there are some nuances. Let's see which licenses can be converted to a new type and how to do it.

> ## How to renew or upgrade a legacy key?

Reviewer: вопрос лишний в конструкциях с how to

> You can renew or upgrade a legacy license in your [AdGuard account](https://adguardaccount.com) or on the [renewal page](https://adguard.com/renew.html).

Reviewer: ```suggestion
You can extend/renew or upgrade a legacy license in your [AdGuard account](https://adguardaccount.com) or on the [Extend or upgrade page](https://adguard.com/renew.html).
```

Reviewer: из редполитики:
- Если подписка действующая и пользователь заранее оплачивает следующий период — *продлить вручную / extend*
- Если подписка отменена или истёк срок действия — *возобновить / renew*

> To renew or update an outdated license in your AdGuard account, go to the *Licenses* tab, find the license you need, and select one of the available options.

Reviewer: ```suggestion
To extend/renew or upgrade a legacy license in your AdGuard account, go to the *Licenses* tab, find the license you need, and select one of the available options.

Reviewer: почему решила не наживать commit suggestion?

> ![Renew or upgrade a license in your AdGuard account](https://cdn.adtidy.org/blog/new/zl9t9legacy2.png)

Reviewer: комментарий по блюру тот же, стрелочки кажутся лишними

> Or, go to the [renewal page](https://adguard.com/renew.html), enter your license, and click *Apply*. On the new page select one of the available options.

Reviewer: ```suggestion
Or go to the [Extend or upgrade page](https://adguard.com/renew.html), enter your license, and click *Apply*. On the new page, select one of the available options.

Reviewer: и тут тот же вопрос

### Licenses for 1/2/3 devices

> **Yearly** licenses for 1, 2, 3 devices can be upgraded to either Personal or Family:

Reviewer: фраза Артёма из задачи: «Как минимум фраза «Годовые лицензии на 1, 2, 3 устройства могут быть сконвертированы как в Персональную» неверна: 3 устройства — это уже и так Персональная лицензия, её не сконвертировать»

Author: да, но там есть мобильные на три устройства. они так и оставались только для мобилки, поэтому их еще можно сконвертировать в Персональную

Reviewer: @Martynuk всё так?

Reviewer: @NataKiss тогда давай так
```suggestion
Mobile Yearly licenses for 3 devices can be converted to Personal licenses or upgraded to Family ones:
```

> ![Renew Yearly licenses for 1, 2, 3 devices](https://cdn.adtidy.org/blog/new/n6sgrpersonal3.png)

Reviewer: комментарий по блюру тот же, стрелочки кажутся лишними

Reviewer: + в описании скриншота ренью, а стрелками указываем на кнопки апгрейд

> If you have a **Standard license for 3 devices**, it already works as a **Personal** license.

> >> If you renew your **Mobile license** for 3 devices, you will be able to use it on a **different set of device**s, not just phones.

Reviewer: `>>` — а что это за выделение?

You can also upgrade your license to Lifetime:

> ![Upgrade to Lifetime](https://cdn.adtidy.org/blog/new/xr2btforever4.png)

Reviewer: комментарий по блюру тот же, стрелочки кажутся лишними

**Lifetime** licenses can only be converted to Family (9 devices) due to technical limitations.

### Licenses for 4+ devices (up to 9)

> **Yearly** licenses for 4 or more devices can be converted to Family licenses, but **Lifetime** licenses cannot be upgraded due to technical limitations.

Reviewer: надо определиться с терминологией converted/upgraded

To renew or update an outdated license in your AdGuard account, go to the *Licenses* tab, find the license you need, and select one of the available options.

> ![Renew or upgrade a license in your AdGuard account](https://cdn.adtidy.org/blog/new/rf4zgklicense2.png)

Reviewer: вот, теперь хороший скрин

Or, go to the [renewal page](https://adguard.com/renew.html), enter your license, and click *Apply*. On the new page select one of the available options.

### Licenses for 1/2/3 devices

**Yearly** licenses for 1, 2, 3 devices can be upgraded to either Personal or Family:

![Upgrade Yearly licenses for 1, 2, 3 devices](https://cdn.adtidy.org/blog/new/fh5r7upgrade3.png)

> If you have a **Standard license for 3 devices**, it already works as a **Personal** license.
> If you renew your **Mobile license** for 3 devices, you will be able to use it on a **different set of device**s, not just phones.

You can also upgrade your license to Lifetime:

![Upgrade to Lifetime](https://cdn.adtidy.org/blog/new/7j4l6gupgrade4.png)

**Lifetime** licenses can only be upgraded to Family (9 devices) due to technical limitations.

### Licenses for 4+ devices (up to 9)

> **Yearly** licenses for 4 or more devices can be upgraded to Family licenses, but **Lifetime** licenses cannot be upgraded due to technical limitations.

Reviewer: так, какая-то путаница
выше пишем: **Lifetime** licenses cannot be upgraded due to technical limitations.
а тут: **Lifetime** licenses can only be upgraded to Family (9 devices) due to technical limitations.

Author: но те для 1,2,3х устройств, а для 4+ вот так. Если вечная на 4+ устройств, то ее некуда улучшать, а на 1 2 3 устройства еще можно до семейной улучшить

Reviewer: а, сорян, проглядела
